clang/include/clang/Driver/Options.td

```diff
@@ -8549,7 +8549,10 @@ def dxc_hlsl_version : Option<["/", "-"], "HV", KIND_JOINED_OR_SEPARATE>,
                      Group<dxc_Group>,
                      Visibility<[DXCOption]>,
                      HelpText<"HLSL Version">,
-                     NormalizedValues<["2016", "2017", "2018", "2021", "202x"]>;
+                     Values<"2016, 2017, 2018, 2021, 202x">,
+                     NormalizedValues<["lang_hlsl2016", "lang_hlsl2017", "lang_hlsl2018", "lang_hlsl2021", "lang_hlsl202x"]>,
+                     NormalizedValuesScope<"LangStandard">,
+                     MarshallingInfoEnum<LangOpts<"HLSLVersion">, "lang_hlsl2021">;
```

Comment:
Do we need a default value for this?
The default language version is set elsewhere to HLSL 2021, having a default here will make it so that we need to set the default in two places instead of one when we change it.